### PR TITLE
docs: added screenshots

### PR DESCRIPTION
**Pull request by matryer**

### CODE REVIEW

File: docs/sources/access/_index.md [hunk 1]
```diff
@@ -71,3 +71,7 @@ This will download the https://github.com/grafana/explore-logs/blob/main/scripts
 That shell file will download some configuration files into your `/tmp/explore-logs` directory and start the docker containers via `docker compose` from there.
 
 Once the docker container has started, navigate to `http://localhost:3000/a/grafana-lokiexplore-app/explore` to access Explore Logs.
+
+# Having trouble?
```
Comment (matyax) [inline docs/sources/access/_index.md:75]: Maybe?

```suggestion
# Having troubles?
```

**Review by matyax — APPROVED** [review]

File: docs/sources/access/_index.md (hunk 1 above)
Comment (matyax) [reply docs/sources/access/_index.md:75]: _Troubleshooting_ sounds great 👍

File: docs/sources/get-started/_index.md [hunk 2]
```diff
@@ -49,3 +51,5 @@ You can modify your time range in two ways:
 ### What do you think?
 
 Please [share your feedback](https://forms.gle/1sYWCTPvD72T1dPH9) and help make Explore Logs better.
+
+![Screenshot of Explore Logs landing page](../images/explore-logs-features.jpeg)
```
Comment (JStickler) [inline docs/sources/get-started/_index.md:55]: I think this screenshot is too busy for documentation.  It's the sort of thing I'd expect to see in a blog post or marketing,  not user documentation.

File: docs/sources/_index.md [hunk 3]
```diff
@@ -31,20 +35,19 @@ Explore Logs is currently in [public preview](/docs/release-life-cycle/). Grafan
 
 # Explore Logs documentation
 
-## Welcome to our new logs experience
+![A new way to look at your logs](images/explore-logs-hero-banner.png)
+![Screenshot of Explore Logs landing page](images/explore-logs-features.jpeg)
```
Comment (JStickler) [inline docs/sources/_index.md:39]: I think this screenshot is too busy for documentation.  It's the sort of thing I'd expect to see in a blog post or marketing,  not user documentation.

**Pull request merged**